cfn.yaml

```diff
@@ -36,8 +36,8 @@ Mappings:
       TableReadCapacity: 25
       TableWriteCapacity: 25
     PROD:
-      TableReadCapacity: 25
-      TableWriteCapacity: 25
+      TableReadCapacity: 75
+      TableWriteCapacity: 4500
```

Comment:
Not sure about putting that in the cloudformation. I'm happy with it being bumped during the migration but the normal state should be a tad bit lower

Reply:
OK. Got it